src/Sparrow.Server/Platform/Pal.cs

```diff
@@ -34,7 +34,18 @@ static Pal()
             }
             else if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
             {
-                fromFilename = Environment.Is64BitProcess ? $"{toFilename}.win.x64.dll" : $"{toFilename}.win.x86.dll";
+                var win7 = "";
+                var osdesc = RuntimeInformation.OSDescription.Split(new[] { "Windows " }, StringSplitOptions.RemoveEmptyEntries);
+                if (osdesc.Length > 1)
+                {
```

Comment:
Is there no other way to do this?

Reply:
OSDescription can give you Windows OS version in dotnet core.. We can parse it in many ways, but I am not familiar with another way to get the OS version in non-string corefx call